topics/clusterwide-config.md

```diff
@@ -0,0 +1,205 @@
+# Clusterwide configuration
+
+Cartridge orchestrates a distributed system of Tarantool instances - a
+cluster. One of the core concepts is a **clusterwide configuration**.
+Every instance in cluster stores a copy of it.
+
+Clusterwide configuration contains the options that must be identical on
+every cluster node, such as topology of the cluster, failover and vshard
+configuration, authentication parameters and ACLs, and user-defined
+configuration.
+
+Clusterwide configuration doesn't provide instance specific parameters:
+ports, workdirs, memory settings etc.
+
+## Internal representation
+
+On file system clusterwide configuration is represented by a **file
+tree**. Inside `workdir` of any configured instance one can find the
+following directory:
+
+```text
+config/
```

Comment:
I think we should add `.tarantool.cookie` here.

Reply:
It's not a part of clusterwide configuration.